tf-psa-crypto/include/psa/crypto.h

```diff
@@ -5528,7 +5527,7 @@ psa_status_t psa_generate_key_iop_complete(
  *                              psa_generate_key_iop_abort() after the
  *                              operation has already been terminated by a call
  *                              to \c psa_generate_key_iop_abort() or
- *                              psa_generate_key_iop_complete() is safe.
+ *                              \c psa_generate_key_iop_complete() is safe.
```

Comment:
Typo: `psa_generate_key_iop_complete()` -> `psa_export_public_key_iop_complete`
(I am not sure if I saw this at an earlier commit or not, sorry if duplicate.)

Reply:
This is the documentation of` psa_generate_key_iop_abort()` so it should be `psa_generate_key_iop_complete() `I just noticed that it was missing `\c `so I added it [here](https://github.com/Mbed-TLS/mbedtls/pull/9778/commits/5fc76a91c69d9566e84cc0752308d4ebfc181e43)

Reply:
Oh, indeed